[iOS] REGRESSION(251612@main): fullscreen close/PiP buttons are too close to the top of the screen

dcrousso opened the pull request:
#### c60ef5766625b753e2c91d81673fe513a147902b
<pre>
[iOS] REGRESSION(251612@main): fullscreen close/PiP buttons are too close to the top of the screen
<a href="https://bugs.webkit.org/show_bug.cgi?id=242361">https://bugs.webkit.org/show_bug.cgi?id=242361</a>
&lt;rdar://problem/92740201&gt;

Reviewed by Tim Horton.

* Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm:
(-[WKFullScreenViewController loadView]):

Canonical link: <a href="https://commits.webkit.org/252157@main">https://commits.webkit.org/252157@main</a>
</pre>

hortont424 commented on Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm:
```diff
@@ -402,7 +402,7 @@ - (void)loadView
     if (fullScreenControlDesignEnabled)
```
While you're here, can you rename this local to have the word "alternate" at the front?

dcrousso committed:
[iOS] REGRESSION(251612@main): fullscreen close/PiP buttons are too close to the top of the screen
https://bugs.webkit.org/show_bug.cgi?id=242361
<rdar://problem/92740201>

Reviewed by Tim Horton.

* Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm:
(-[WKFullScreenViewController loadView]):

Canonical link: https://commits.webkit.org/252157@main
